client/my-sites/controller.js

```diff
@@ -71,7 +71,8 @@ function createNavigation( context ) {
 	}
 
 	return (
-		<NavigationComponent path={ context.path }
+		<AsyncLoad require="my-sites/navigation"
```

Comment:
> (async-load-my-sites-picker) 6.1 kB

Maybe we bundle a big part of sites picker in the main build? Not sure. It's way too small.